tests/tracer/test_tracer.py

```diff
@@ -1934,3 +1935,36 @@ def test_ctx_api():
         _context.get_item("appsec.key")
     with pytest.raises(ValueError):
         _context.get_items(["appsec.key"])
+
+
+def test_installed_excepthook():
+    ddtrace.install_excepthook()
+    assert sys.excepthook is ddtrace._excepthook
+    ddtrace.uninstall_excepthook()
+    assert sys.excepthook is not ddtrace._excepthook
+    ddtrace.install_excepthook()
+    assert sys.excepthook is ddtrace._excepthook
+
+
+def test_excepthook():
+    class Foobar(Exception):
+        pass
+
+    called = {}
+
+    def original(tp, value, traceback):
+        called["yes"] = True
+
+    sys.excepthook = original
+    ddtrace.install_excepthook()
+
+    e = Foobar()
+
+    tracer = ddtrace.Tracer()
+    tracer._writer.dogstatsd = mock.Mock()
+    with override_global_tracer(tracer):
+        sys.excepthook(e.__class__, e, None)
+    tracer._writer.dogstatsd.increment.assert_has_calls(
+        (mock.call("datadog.tracer.uncaught_exceptions", 1, tags=["class:Foobar"]),)
+    )
+    assert called
```

Comment:
```suggestion
    assert called["yes"]
```

?

Reply:
Actually, I removed the dogstatsd incrementing, from the excepthook. Now instead it creates the app_started event. 

`if asbool(os.getenv("DD_INSTRUMENTATION_TELEMETRY_ENABLED", True)):
            telemetry_writer.add_error(1, tp)
            telemetry_writer.enable()`
            
 Therefore I think that the testing done in test_telemetry.py should cover this.